content/sensu-go/5.9/reference/hooks.md

```diff
@@ -42,6 +42,34 @@ See the [check specification][6] to configure the `check_hooks` attribute.
 The hook command output, status, executed timestamp and duration are captured
 and published in the resulting event.
 
+You can use `sensuctl` to view this data:
+
+{{< highlight shell >}}
+sensuctl event info entity_name check_name --format yaml
+{{< /highlight >}}
+
+{{< highlight shell >}}
+type: Event
+api_version: core/v2
+metadata:
+  namespace: default
+spec:
+  check:
+    "...": "..."
```

Comment:
```suggestion
    ...
```

Reply:
Oh weird, I guess the converter I used doesn't know what to do with that.